eda/tests/unittests/resources/adult/test_data.csv

```diff
@@ -0,0 +1,201 @@
+age,workclass,fnlwgt,education,education-num,marital-status,occupation,relationship,race,sex,capital-gain,capital-loss,hours-per-week,native-country,class
```

Comment:
Have you considered put these csv fils in s3 and load them like what we are doing in other modules' unit tests?
Ditto for the rest

Reply:
Then you can't run these tests offline (like when I was working when traveling).